pkg/libvirttools/virtualization.c

```diff
@@ -20,16 +20,64 @@ limitations under the License.
 #include "virtualization.h"
```

Comment:
Yes, I agree regarding images and for cases when as you wrote " I try to avoid writing huge functions in Go code which would contain a lot of C.something statements", I asked only regarding this one as here there is no much logic and it seems to me opposite for this case, that the volume of code could be less with direct usage of C.something and  even less complex without additional wrapper. And regarding const char*, you can safely pass to function just char * () without explicit cast using C.CString.